<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>901</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CH01</string>
     </attribute>
     <widget class="QwtPlot" name="qwt_ch1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>871</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtPlot" name="qwt_ch1_fft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>871</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>CH02</string>
     </attribute>
     <widget class="QwtPlot" name="qwt_ch2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>871</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtPlot" name="qwt_ch2_fft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>871</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>CH03</string>
     </attribute>
     <widget class="QwtPlot" name="qwt_ch3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>871</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtPlot" name="qwt_ch3_fft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>871</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>CH04</string>
     </attribute>
     <widget class="QwtPlot" name="qwt_ch4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>871</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtPlot" name="qwt_ch4_fft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>871</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>311</width>
      <height>602</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>网络配置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>本地IP：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_host_ip">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>服务器IP：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_server_ip">
            <property name="text">
             <string>192.168.1.176</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>端口号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_port_num">
            <property name="text">
             <string>8322</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_set">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_disconnect">
            <property name="text">
             <string>断开连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_close_remote">
            <property name="text">
             <string>关闭远程服务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_close_test">
            <property name="text">
             <string>测试按钮</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>控制信息配置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>频率(KHz):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_freq">
            <property name="text">
             <string>50</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_freq_set">
            <property name="text">
             <string>设定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>采样率kHz):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_fs">
            <property name="text">
             <string>100</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_fs_set">
            <property name="text">
             <string>设定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>增益(1,2): </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_gain_1">
            <property name="text">
             <string>40</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_gain_2">
            <property name="text">
             <string>40</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_gain_set">
            <property name="text">
             <string>设定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButton_sample">
            <property name="text">
             <string>启动采样</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_close_sample">
            <property name="text">
             <string>关闭采样</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clear">
          <property name="text">
           <string>清空接收区</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1263</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_disconnect</tabstop>
  <tabstop>pushButton_set</tabstop>
  <tabstop>lineEdit_host_ip</tabstop>
  <tabstop>lineEdit_port_num</tabstop>
  <tabstop>lineEdit_server_ip</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
